app/experimenter/docs/adrs/0005-doc-hub.md

```diff
@@ -0,0 +1,231 @@
+
+The documentation that will be part of the hub includes:
+
+- A new front-page or home-page doc, likely written in MD, with a brief overview of Nimbus and instructions for users on the workflow for doc maintenance
+- New docs, likely written in MD, and depending on the chosen option, possibly optionally with React components (if Docusaurus or Storybook) or custom JS (if Docusaurus, Storybook, or `mdBook`)
+  - For Fenix engineers: SDK docs for integration in Fenix
+  - For experiment owners: How to use the console, either living here or in Mana
+  - For data scientists: How to add Jetstream configs
+  - How to read automated analysis, either living here or in Mana
+  - In the future, iOS and Gecko integration docs
+- Existing docs
+  - For Desktop engineers: [SDK docs for integration in desktop front end](https://www.notion.so/k88hudson/Nimbus-Migration-Guide-for-Desktop-Front-End-Experiments-d36c21e505f84550aad1202897fc4ba3)
+- Various Google docs and Mana links
+- Possibly existing MD docs living in other repos
+
+## Decision Drivers
+
+- Docs should be kept in a central location and cross-linking should be easy
+- Finding relevant documentation should be painless, quick, and clear for all of our users, and cover their specific doc needs, like copying a code snippet in various languages
+- Docs should be easily editable by anyone; maintenance and workflow should be straightforward and scalability should be taken into account
+- Set up, hosting, and deployment efforts should be minimal
+
+## Considered Options
+
+- Mana / Confluence
+- Docusaurus + GH Pages
+- Experimenter repo Wiki
+- Storybook
+- mdBook + GH Pages
+
+## Decision Outcome
+
+Chosen option: "Docusaurus + GH pages", because Docusaurus provides a flexible and easily updated sidebar, allows MD and custom JS when needed, and provides a robust search bar. GitHub Pages takes care of deployment efforts.
+
+### Positive Consequences
+
+- Documents will be written in MD, meaning most edits can be made in the GH interface
+- Provides an easy-to-use sidebar, Docusaurus automatically imports and reflects MD file links with a denotation at the top of MD files, and also allows external links (like to Mana) to live in the sidebar
+- Has a great search bar
+- Is flexible and allows custom JS (and React components if needed) to allow a tabbed section to copy code snippets in various languages
+- Sets us up for easy scalability beyond our current needs
+- GH Pages will handle deployment
+- Tiny win: GH Pages allows custom domains, if we later want to use a nicer URL
+
+### Negative Consequences
+
+- Requires time invested in set up
+- The GH and PR workflow requires a couple more steps than the Mana or GH Wiki option workflow
+  - Note: any friction will be smoothed out as much as possible with thorough "how to maintain the docs" documentation
+- Allowing custom JS/React increases the complexity of the platform
```

Comment:
I'm still confused why the duplication of pros/cons up here, I don't think we do this in other ADRs. Not a big deal though. 🤷

Reply:
Hmm, maybe I'll go ahead and remove them. These sections are "optional" according to our template and you're right that it's perhaps superfluous.